src/data-types/interfaces.ts

```diff
@@ -43,29 +44,33 @@ export type TxMAEgressEventName =
   | 'AIS_EVENT_TRANSITION_APPLIED'
   | 'AIS_EVENT_TRANSITION_IGNORED'
   | 'AIS_EVENT_IGNORED_STALE'
-  | 'AIS_EVENT_IGNORED_IN_FUTURE'
-  | 'AIS_EVENT_IGNORED_ACCOUNT_DELETED';
+  | 'AIS_EVENT_IGNORED_ACCOUNT_DELETED'
+  | 'AIS_EVENT_IGNORED_IN_FUTURE';
 
```

Comment:
I'm wondering if `TxMAEgressEventName` would work better as an enum.

Reply:
That could work, but I'd do that as part of another ticket. We could raise it as a tech debt ticket